packages/gnutls/SPECS/gnutls.spec

```diff
@@ -100,38 +159,71 @@ echo "SYSTEM=NORMAL" >> tests/system.prio
 # via the crypto policies
 
 %build
-export CCASFLAGS="$CCASFLAGS -Wa,--generate-missing-build-notes=yes"
-
-./configure --prefix=/usr/sgug \
-   --disable-option-checking \
-   --disable-doc \
-   --disable-hardware-acceleration \
-   --disable-padlock \
-   --disable-strict-der-time \
-   --disable-heartbeat-support \
-   --disable-tests \
-   --disable-full-test-suite \
-   --disable-gcc-warnings \
-   --disable-libdane \
-   --disable-nls \
-   --disable-rpath \
-   --enable-sha1-support \
-   --disable-static \
-   --disable-non-suiteb-curves \
-   --disable-guile \
-   --with-default-trust-store-pkcs11="pkcs11:" \
-   --libdir=%{?__isa_bits==64:/usr/sgug/lib}%{!?__isa_bits==32:/usr/sgug/lib32} \
-   --prefix=/usr/sgug
+CCASFLAGS="$CCASFLAGS -Wa,--generate-missing-build-notes=yes"
+export CCASFLAGS
+
+## These should be checked by m4/guile.m4 instead of configure.ac
+## taking into account of _guile_suffix
+#guile_snarf=%{_bindir}/guile-snarf2.2
+#export guile_snarf
+#GUILD=%{_bindir}/guild2.2
+#export GUILD
+
+export CPPFLAGS="-I%{_includedir}/libdicl-0.1 -DLIBDICL_NEED_GETOPT=1 -D_SGI_SOURCE -D_SGI_REENTRANT_FUNCTIONS"
+export LDFLAGS="-ldicl-0.1 $RPM_LD_FLAGS"
+
+# Hints to autoconf so that some gnulib tests pass
+export gl_cv_func_select_supports0=no
+export gl_cv_func_select_detects_ebadf=no
+
+%configure --with-libtasn1-prefix=%{_prefix} \
+%if %{with fips}
+           --enable-fips140-mode \
+%endif
+	   --enable-sha1-support \
+           --disable-static \
+           --disable-openssl-compatibility \
+           --disable-non-suiteb-curves \
+           --with-default-trust-store-pkcs11="pkcs11:" \
+           --htmldir=%{_docdir}/manual \
+%if %{with guile}
+           --enable-guile \
+           --with-guile-extension-dir=%{_libdir}/guile/2.2 \
+%else
+           --disable-guile \
+%endif
+%if %{with dane}
+           --with-unbound-root-key-file=/var/lib/unbound/root.key \
+           --enable-dane \
+%else
+           --disable-dane \
+%endif
+           --disable-rpath
+
+# Unused on sgug
+#           --with-system-priority-file=%{_sysconfdir}/crypto-policies/back-ends/gnutls.config \
```

Comment:
These are commented out _and also moved out of the `configure` block - because of the backslashes at the end of the lines.